src/wasm-binary.h

```diff
@@ -948,13 +949,18 @@ class WasmBinaryWriter {
   void writeExports();
   void writeDataCount();
   void writeDataSegments();
+  void writeEvents();
 
   // name of the Function => index. first imports, then internals
   std::unordered_map<Name, Index> mappedFunctions;
   // name of the Global => index. first imported globals, then internal globals
   std::unordered_map<Name, uint32_t> mappedGlobals;
+  // name of the Event => index. first imported events, then internal events
+  std::unordered_map<Name, uint32_t> mappedEvents;
+
```

Comment:
separate from this PR, I think this might overlap with the `BinaryIndexes` stuff? Could be a refactoring opportunity.

Reply:
Will do in a separate PR.